docs/commands/lint.md

```diff
@@ -191,6 +231,8 @@ redocly lint --max-problems 200
 < ... 2 more problems hidden > increase with `--max-problems N`
 ```
 
+The default value for `max-problems` is 100.
```

Comment:
We usually report default values in the summary tables (should this move into line 34?).

Reply:
I'll move it.